packages/tenant-process/src/services/readModelService.ts

```diff
@@ -31,6 +28,64 @@ async function getTotalCount(
   );
   throw ErrorTypes.GenericError;
 }
-*/
 
-export const readModelService = {};
+function listTenantsFilters(
+  name: string | undefined
+): [Filter<{ data: Tenant }>, Filter<{ data: Tenant }>] {
```

Comment:
Update the branch and use the new `MongoDBFilter` type here

Reply:
`MongoDBFilter` not yet available on this branch